src/libs/OptionsListUtils.ts

```diff
@@ -502,14 +507,32 @@ function getAllReportErrors(report: OnyxEntry<Report>, reportActions: OnyxEntry<
         const transaction = allTransactions?.[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`];
         if (TransactionUtils.hasMissingSmartscanFields(transaction ?? null) && !ReportUtils.isSettled(transaction?.reportID)) {
             reportActionErrors.smartscan = ErrorUtils.getMicroSecondOnyxErrorWithTranslationKey('iou.error.genericSmartscanFailureMessage');
+            reportAction = undefined;
```

Comment:
NAB: Why undefined instead of calling this like below? What am I missing?
```
ReportUtils.getReportActionWithMissingSmartscanFields
```

Reply:
This code can only be reached by checking the transaction object, while the if below looks at report actions.

So this smart scan error is related to the transaction and not a report action, hence I set it as undefined